Anature/src/application/enums/MoveIds.java

```diff
@@ -2,7 +2,9 @@
 
 public enum MoveIds
 {
-	NullMove, Tackle, Grumble, Flamethrower, Double_Punch, Skip_Turn, Pocket_Sand, Flail,
+	NullMove, Tackle, Grumble, Flamethrower, Double_Punch, Skip_Turn, Pocket_Sand, Flail, Cinder, Smash, Smog_Wave,
+	Fire_Cell, Fire_Torture ,Scary_Face, Flame_Bout, Electro_Sonic, Stormy_Breeze, Holler, Light_Missle, Lightning, Electrocution, 
+	Magical_Spice, Forgotten_Awakening, Healing_Winds, Clop, Antler_Shot, Leen
```

Comment:
Could you organize the moves by type? Makes it easier to identify what each move is.